src/php/Filesystem/Domain/FakeFilesystem.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Filesystem\Domain;
+
+final class FakeFilesystem implements FilesystemInterface
```

Comment:
In my opinion, the word `fake` is more related to testing.

In fact, you are using the `Null Object Design Pattern`, I think maybe make more sense to rename the class from `FakeFileSystem` to  `NullObjectFileSystem` or `NullFileSystem`.

Reply:
I will rename it to `NullFileSystem`